client/src_editor/Editor_Blocks_Refmt.re

```diff
@@ -0,0 +1,56 @@
+open Editor_Types;
+
+let refmtAsLangSibling = (code, lang, blockId, callback) =>
+  switch (lang) {
+  | RE =>
+    Js.Promise.(
+      Editor_Worker.mlToRe(. code)
+      |> then_(
+           (
+             result: Belt.Result.t(string, Worker_Evaluator.Types.Refmt.error),
+           ) => {
+           result
+           |> (
+             fun
+             | Ok(re) => callback(re, blockId)
+             | Error(error) => Js.log(error)
```

Comment:
I added a toast in master. Could you please rebase and add it? Notify.error(a string)

Reply:
sounds good, was meaning to ask you how you'd like to handle errors like these